src/integration-test/integration/MappingsClient.test.ts

```diff
@@ -0,0 +1,485 @@
+    };
+    let property = await mappingsClient.createGroupProperty(accessToken, testIModel.id, mappingIds[0], groupId, newProperty);
+    expect(property).to.not.be.undefined;
+    groupPropertyId = property.id;
+
+    newProperty.propertyName = "prop2";
+    property = await mappingsClient.createGroupProperty(accessToken, testIModel.id, mappingIds[0], groupId, newProperty);
+    expect(property).to.not.be.undefined;
+
+    newProperty.propertyName = "prop3";
+    property = await mappingsClient.createGroupProperty(accessToken, testIModel.id, mappingIds[0], groupId, newProperty);
+    expect(property).to.not.be.undefined;
+
+    //create calculated properties
+
+    const newCalculatedProperty: CalculatedPropertyCreate = {
+      propertyName: "calc1",
+      type: CalculatedPropertyType.Length,
+    };
+    let calcProperty = await mappingsClient.createCalculatedProperty(accessToken, testIModel.id, mappingIds[0], groupId, newCalculatedProperty);
+    expect(calcProperty).to.not.be.undefined;
+    calculatedPropertyId = calcProperty.id;
+
+    newCalculatedProperty.propertyName = "calc2";
+    calcProperty = await mappingsClient.createCalculatedProperty(accessToken, testIModel.id, mappingIds[0], groupId, newCalculatedProperty);
+    expect(calcProperty).to.not.be.undefined;
+    
+    newCalculatedProperty.propertyName = "calc3";
+    calcProperty = await mappingsClient.createCalculatedProperty(accessToken, testIModel.id, mappingIds[0], groupId, newCalculatedProperty);
+    expect(calcProperty).to.not.be.undefined;
+
+    //create customCalculations
+
+    const newCustomCalculation: CustomCalculationCreate = {
+      propertyName: "cust1",
+      formula: "1+1",
+      quantityType: QuantityType.Distance,
+    };
+    let custCalculation = await mappingsClient.createCustomCalculation(accessToken, testIModel.id, mappingIds[0], groupId, newCustomCalculation);
+    expect(custCalculation).to.not.be.undefined;
+    customCalculationId = custCalculation.id;
+
+    newCustomCalculation.propertyName = "cust2";
+    custCalculation = await mappingsClient.createCustomCalculation(accessToken, testIModel.id, mappingIds[0], groupId, newCustomCalculation);
+    expect(custCalculation).to.not.be.undefined;
+
+    newCustomCalculation.propertyName = "cust3";
+    custCalculation = await mappingsClient.createCustomCalculation(accessToken, testIModel.id, mappingIds[0], groupId, newCustomCalculation);
+    expect(custCalculation).to.not.be.undefined;
+  });
+
+  after(async () => {
+    let response: Response;
+    while(mappingIds.length > 0) {
+      response = await mappingsClient.deleteMapping(accessToken, testIModel.id, mappingIds.pop() ?? "");
+      expect(response.status).to.be.eq(204);
+    }
+    await testIModelGroup.cleanupIModels();
+  });
+
+  //mapping tests
+  it("General - get all with iterator", async function () {
+    const iterator = mappingsClient.getMappingsIterator(accessToken, testIModel.id);
+    let mapping = await iterator.next();
```

Comment:
Use the `for await` syntax instead of `next/done`.

Reply:
changed syntax to for await